src/implementations/vanilla/components/banner/_banner.scss

```diff
@@ -107,28 +112,28 @@ $_font-weight: null !default;
 /* stylelint-disable-next-line order/order */
 @include breakpoints.up('l') {
   // medium banner - ratio 4:1
-  .ecl-banner::before,
-  .ecl-banner--m::before {
+  .ecl-banner,
+  .ecl-banner--m {
+    .ecl-banner__image {
+      aspect-ratio: 4/1;
+    }
+  }
+
+  .ecl-banner--plain-background::before {
```

Comment:
do we still need this? I think that it was for the ratio

Reply:
But nothing has changed regarding the plain-background variant, there is no image so the aspect-ratio is still set by this css hack.